.github/workflows/versions.yml

```diff
@@ -0,0 +1,114 @@
+name: Verify distributed versions
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: Tag name for WordPress release
+        required: true
+        type: string
+      package:
+        description: Package for WordPress release
+        required: true
+        type: string
+
+permissions: {}
+
+jobs:
+  verify:
+    name: "${{ inputs.package }} ${{ inputs.tag }}"
+    runs-on: ubuntu-latest
+    permissions: {}
+    timeout-minutes: 10
+    steps:
+      - name: Prepare tag name
+        id: tag
+        uses: johnbillion/rave-wordpress/.github/actions/prepare-tag-name@verify-version-offers
+        with:
+          tag: ${{ inputs.tag }}
+
+      - name: Verify versions distributed by GitHub
+        if: inputs.package == 'github-zip'
+        env:
+          TAG: ${{ steps.tag.outputs.short }}
+        run: |
+          set -e
+          latest_tag=$(curl -s https://api.github.com/repos/WordPress/WordPress/tags | jq -r '.[0].name')
+          if [ "$latest_tag" != "$TAG" ]; then
+            echo "Error: Found a newer tag ($latest_tag) than $TAG"
+            exit 1
+          fi
+
+      - name: Verify versions distributed by roots/wordpress-full
+        if: inputs.package == 'roots-wordpress-full'
+        env:
+          TAG: ${{ steps.tag.outputs.short }}
+        run: |
+          set -e
+          latest_version=$(curl -s https://repo.packagist.org/p2/roots/wordpress-full.json | jq -r '.packages["roots/wordpress-full"] | map(.version) | .[]' | grep -Ev -- '-(RC|rc|beta|alpha|dev)' | sort -V | tail -n1)
+          if [ "$latest_version" != "$TAG" ]; then
+            echo "Error: Found a newer version ($latest_version) than $TAG"
+            exit 1
+          fi
+
+      - name: Verify versions distributed by johnpbloch/wordpress
+        if: inputs.package == 'johnpbloch-wordpress'
+        env:
+          TAG: ${{ steps.tag.outputs.short }}
+        run: |
+          set -e
+          latest_version=$(curl -s https://repo.packagist.org/p2/johnpbloch/wordpress.json | jq -r '.packages["johnpbloch/wordpress"] | map(.version) | sort_by(.) | last')
+          if [ "$latest_version" != "$TAG" ]; then
+            echo "Error: Found a newer version ($latest_version) than $TAG"
+            exit 1
+          fi
+
+      - name: Verify versions distributed by Docker
+        if: inputs.package == 'docker-wordpress'
+        env:
+          TAG: ${{ steps.tag.outputs.long }}
+        run: |
+          set -e
+          latest_tag=$(curl -s 'https://hub.docker.com/v2/repositories/library/wordpress/tags?page_size=100' | jq -r '.results[].name' | grep -E '^[0-9]\.[0-9]\.[0-9]$' | sort -V | tail -n1)
```

Comment:
@johnbillion Could it be that tis regexp will miss two digit versions? e.g. `4.19.1`